tests/security/cc/ipsec_client.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run CC 'ipsec' client case
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#101226
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test;
+use Utils::Architectures;
+use lockapi;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    my $server_ip = get_var('SERVER_IP', '10.0.2.101');
+    my $client_ip = get_var('CLIENT_IP', '10.0.2.102');
+
+    # We don't run setup_multimachine in s390x, but we need to know the server and client's
+    # ip address, so we add a known ip to eth0.
+    if (is_s390x) {
+        assert_script_run("ip addr add $client_ip/24 dev eth0");
+    }
+
+    mutex_wait('IPSEC_SERVER_READY');
+
+    assert_script_run("cd $audit_test::test_dir/ipsec_configuration/toe");
+
+    # Setup the ipip tunnel to the IPSec gateway and test it
+    assert_script_run("./ipsec_setup_tunnel_toe.sh start $client_ip $server_ip");
+    assert_script_run('ping -W1 -c1 192.168.100.1');
```

Comment:
Which file or script define 192.168.x.x ip addr, please add some comments?

Reply:
@Amrysliu 
If my understanding is correct, can we remove the line like ping -W1 192.xx?
Since ip sec test should fail if the ip can not ping there. it is only debug purpose and may lead to some confusion why ping so many undefined ip addrs

Reply:
I prefer to keep it. I didn't test if this line could be removed.  And if it fails in `ping -W1 192.x`, we could easy to get the problems.

Reply:
Then no problem, however, since there are several lines we need ping different remote ip addrs.
Can you please update the comments like below:
"In ipsec_setup_tunnel_server.sh script, it will set up several ip addresses like 192.168.x.x. we may need check them are accessible to find any network problems easily"